src/containers/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ConnectedBooksList from '../components/BooksList';
+import BooksForm from '../components/BooksForm';
+
+const App = () => (
+  <div>
+    <ConnectedBooksList />
+    <BooksForm />
+  </div>
+);
+
+export default App;
```

Comment:
App component should be a presentational component(placed in the `components` directory since it does not access the state.
Read more about presentational components and container components [here](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0).

Reply:
App.js has now been returned to the components directory